docs/data/joy/getting-started/quick-start/quick-start.md

```diff
@@ -0,0 +1,67 @@
+# Quick start
+
+<p class="description">Get started with Joy, a simple and beautiful starting point for your design system, in 3 quick steps.</p>
+
+## 1. Installation
+
+Install Joy via either npm or yarn using the commands below in your directory of choice.
+
+### npm
+
+```sh
+npm init react-app spark-joy
+cd spark-joy
+npm install @mui/joy @emotion/react @emotion/styled
+npm start
+```
+
+### Yarn
+
+```sh
+yarn create react-app spark-joy
+cd spark-joy
+yarn add @mui/joy @emotion/react @emotion/styled
+yarn start
+```
+
+## 2. Set up the `CssVarsProvider`
```

Comment:
Feel like we're missing a word explaining what `CssVarsProvider` is. 

```suggestion
## 2. Set up the `CssVarsProvider` utility
```

Would that be correct? Something like that.

Reply:
I would not add `utility` here because I feel that it refers to as optional but `CssVarsProvider` is required.

Or to make it more generic, we could use `2. Set up the Provider`.

Reply:
Would that be correct then? 
```suggestion
## 2. Set up the `CssVarsProvider` component
```

Reply:
That works.